http/api.go

```diff
@@ -222,24 +207,16 @@ func HandleBattleSimulate(w http.ResponseWriter, r *http.Request) {
 	type roundResults struct {
 		Transactions []Transaction
 		Ended        bool
+		Parties      []*Party
 	}
 
 	results := roundResults{
 		transactions,
 		ended,
+		hb.Battle.Parties(),
 	}
 
-	bytes, err := json.Marshal(results)
-	if err != nil {
-		log.Printf("Failed to marshal into JSON: %s", err)
-		w.WriteHeader(500)
-		w.Write([]byte(`Internal server error: failed to marshal round results`))
-		return
-	}
-
-	w.Header().Set("Content-Type", "application/json")
-	w.WriteHeader(200)
-	w.Write(bytes)
+	JSONResponse(w, results)
```

Comment:
A good pattern for this is to create a custom type that implements the `http.ResponseWriter` interface so that you can have custom methods like `custom.JSON(status, payload)` or `custom.Error(msg)`.